core/src/test/java/bio/terra/pearl/core/service/export/EnrolleeImportServiceTests.java

```diff
@@ -168,6 +189,52 @@ public void testImportEnrolleeUpdateCSV(TestInfo info) {
         profileExpected2.setId(enrollee2.getProfileId()); //should be same profile
         profileExpected2.setBirthDate(LocalDate.parse("1990-10-10"));
         verifyParticipant(imports.get(1), studyEnvId, userExpected2, enrolleeExpected2, profileExpected2);
+        verifySurvey(dataImportUpdate.getImportItems().get(1), "medical_history", "diagnosis", "not healthy");
+    }
+
+    @Test
+    @Transactional
+    public void testImportEnrolleeKitRequestsCSV(TestInfo info) {
+        String csvStringSingleKit = """
+                column1,column2,column3,account.username,account.createdAt,enrollee.createdAt,profile.birthDate,sample_kit.status,sample_kit.createdAt,sample_kit.sentAt,sample_kit.trackingNumber,sample_kit.sentToAddress,sample_kit.kitType
```

Comment:
to make this more readable, strip out the columns that aren't necessary for kit processing